geo/src/algorithm/voronoi_diagram.rs

```diff
@@ -4,39 +4,77 @@ use geo_types::coord;
 
 use crate::{Coord, EuclideanLength, GeoFloat, Line, MultiPoint, Point, Polygon, Rect, Triangle};
 
-use crate::triangulate_delaunay::{
-    DelaunayTriangle, DelaunayTriangulationError, DEFAULT_SUPER_TRIANGLE_EXPANSION,
-};
+use crate::triangulate_delaunay::{DelaunayTriangle, DelaunayTriangulationError};
 use crate::{BoundingRect, TriangulateDelaunay};
 
 type Result<T> = result::Result<T, VoronoiDiagramError>;
 
+const DEFAULT_CLIPPING_MASK_EXPANSION: f64 = 20.;
```

Comment:
probably want this to be `pub` so users can know the default

Reply:
I applied to the `DEFAULT_SUPER_TRIANGLE_EXPANSION` in `triangulate_delaunay`